code/datums/quirks/negative.dm

```diff
@@ -413,9 +413,11 @@
 	var/slot_string = "limb"
 	medical_record_text = "During physical examination, patient was found to have a prosthetic limb."
 	hardcore_value = 3
+	var/limb_slot // Tegu edit
 
 /datum/quirk/prosthetic_limb/on_spawn()
-	var/limb_slot = pick(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
```

Comment:
Dont delete this, just comment it out, that way TG will write over the commented out code if they change it